docs/index.md

```diff
@@ -0,0 +1,73 @@
+# The SpatialOS Unreal GDK (pre-alpha)
+
+<%(TOC)%>
+
+Welcome to the SpatialOS and Unreal GDK community.
+
+Our SpatialOS community shapes the functionality of the Unreal GDK, so make sure you are part of the conversation. Come and find us on [Discord](https://discordapp.com/channels/311273633307951114/339471548647866368) or talk to us on the **Feedback** section in our [forums](https://forums.improbable.io/) (use the **unreal-gdk** tag - [this link](https://forums.improbable.io/new-topic?category=Feedback&tags=unreal-gdk) takes you there and pre-fills the category and tag). You can also [sign up for SpatialOS Unreal GDK updates](http://go.pardot.com/l/169082/2018-06-15/27ld2t) direct to your mailbox.
+
+#### What's the Unreal GDK all about?
+The **SpatialOS Unreal GDK** is an Unreal Engine 4 (UE4) [module](https://docs.unrealengine.com/en-us/Programming/UnrealBuildSystem/ModuleFiles) made by Improbable. You can use the Unreal GDK to integrate persistent multiplayer worlds into your Unreal game using the SpatialOS platform.
+
+Find out more about the GDK for Unreal in [our blogpost](https://improbable.io/games/blog/spatialos-unreal-gdk-pre-alpha).
+
+You can [sign up for SpatialOS Unreal GDK updates](http://go.pardot.com/l/169082/2018-06-15/27ld2t) direct to your mailbox.
+
+* License: use of the contents of this repository is subject to the [Improbable license]({{urlRoot}}/license)
+* Version: pre-alpha (for evaluation only)
+* Unreal Game Development Kit repository: [github.com/spatialos/UnrealGDK](https://github.com/spatialos/UnrealGDK)
+
+## Recommended use
+We are releasing the GDK in [pre-alpha](https://docs.improbable.io/reference/13.1/shared/release-policy#maturity-stages) so we can react to feedback and iterate on development quickly. To facilitate this, during our pre-alpha stage we don't have a formal deprecation cycle for APIs and workflows. This means that everything and anything can change. In addition, documentation is limited and some aspects of the GDK are not optimized.
+
+Given this, we don’t recommend using the the Unreal GDK for games in the production phase just yet.
+
+**Who is it for?**<br>
+We recommend you start using the GDK if:
+* You don’t mind the rough edges and are interested in contributing to the GDK (see our [contributions]({{urlRoot}}/contributing) policy).
```

Comment:
I think you need a line break above this line, because currently the bullet points aren't working.